_posts/en/pages/2016-01-01-meetings.md

```diff
@@ -5,11 +5,33 @@ lang: en
 title: IRC Meetings
 name: meetings
 permalink: /en/meetings/
-version: 2
+version: 3
 ---
-The project usually holds IRC meetings every Thursday at 19:00 UTC in `#bitcoin-core-dev` on irc.freenode.net.
-Everyone is welcome to attend.
+The project holds several recurring meetings in `#bitcoin-core-dev` on
+irc.freenode.net.  Everyone is welcome to attend.  Logs and
+automatically-generated meeting minutes may be found [here][meetbot].
 
-The following summaries list may not be complete due to constraints on volunteers' time. To view more recent logs, all meeting logs and raw minutes can be found [here](http://www.erisian.com.au/meetbot/bitcoin-core-dev/).
+Current meeting schedule:
 
-{% include meetings.html %}
+- General developer meeting: Thursday 19:00 UTC (every week)
+- Wallet developer meeting: Friday 19:00 UTC (every second week)
```

Comment:
Is there a concise way of saying which alternate week this is on?

Reply:
Not in a particularly useful way that I'm aware of in plain text.  I can probably generate a calendar using a simple table and the site generator's template language.  I'll give that a try today.

Reply:
I don't think this is needed before merging this PR.

Reply:
"Every odd week of the year"?

Reply:
I'm not sure that's well specified enough.  The man page for `date(1)` gives me multiple options

       %U     week number of year, with Sunday as first day of week (00..53)

       %V     ISO week number, with Monday as first day of week (01..53)

Those produce results with different oddness:

    $ date +%U ; date +%V
    32
    33

Also, a year has 52.14 to 52.28 weeks, so I assume oddness can change from year to year.

Reply:
I think it's fine to leave this actually. "If you have any questions about the date or time of an upcoming meeting, please ask in #bitcoin-core-dev on Freenode." seems fine to me.